### PR TITLE
[ENDPOINT] Get /categories OT198-41

### PR DESCRIPTION
**Pull request by honeybadger2788**
[OT198-41](https://alkemy-labs.atlassian.net/browse/OT198-41)
## Description

AS a admin user
I WANT TO get the categories list
SO THAT I can manage categories

### Acceptance criteria
GET to /categories. It will bring the categories list. It should just return the category's name attribute.

## Evidence

- Create /categories endpoint
image

- Create categories route
image

- Fix categories controller
image

- List categories names as a service
image

### CODE REVIEW

**Review by nicobes14 — DISMISSED** [review]
Muy bueno Noe!

File: controllers/categories.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const { Category } = db
 module.exports = {
   list: catchAsync(async (req, res, next) => {
     try {
-      const categories = await Category.findAll()
+      const categories = await Category.findAll({ attributes: ['name'] })
```
Comment (gabodmc) [inline controllers/categories.js:12]: delegar las funcionalidades de la db al servicio

**Review by gabodmc — CHANGES_REQUESTED** [review]
Excelente PR. Muy bien expuesta la evidencia. Solo una correción

**Review by nicobes14 — APPROVED** [review]

**Review by gabodmc — APPROVED** [review]

**Review by HernanGarcia1985 — APPROVED** [review]

**Review by HernanGarcia1985 — APPROVED** [review]
Muy bueno!

**Pull request merged**